core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1279,6 +1273,16 @@ private void updateMediaSession(final PlayerStatus playerStatus) {
             );
         }
 
+        if (UserPreferences.showSleepTimerOnFullNotification()) {
+            sessionState.addCustomAction(
+                    new PlaybackStateCompat.CustomAction.Builder(
+                            CUSTOM_ACTION_TOGGLE_SLEEP_TIMER,
+                            getString(R.string.sleep_timer_label),
+                            R.drawable.ic_sleep
```

Comment:
The icon has to be one from the ui/png-icons folder because these get rasterized on build

Reply:
I moved the icons, but they are still not visible in the notification.

Reply:
You need to use a different copy of the icons. The reason is that these icons automatically switch between black and white depending on the app theme. The icons for the notifications need to have a fixed color in `android:fillColor="..."` because they don't know about the app theme